addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8646,7 +8646,7 @@ msgid "The PVR manager has been enabled without any"
 msgstr ""
 
 msgctxt "#19242"
-msgid "enabled PVR Add-on. Enable at least one Add-on"
+msgid "enabled PVR add-on. Enable at least one add-on"
```

Comment:
we  need to clean up those concatenated strings as we now have a textbox for dialogs instead of separate lines. Something for later on